commands/kubernetes.go

```diff
@@ -912,20 +918,32 @@ func RunKubernetesNodePoolDelete(c *CmdConfig) error {
 	if err != nil {
 		return err
 	}
+	nodeID := c.Args[2]
 
 	force, err := c.Doit.GetBool(c.NS, doctl.ArgForce)
 	if err != nil {
 		return err
 	}
-	if force || AskForConfirm("delete this Kubernetes node pool") == nil {
-		kube := c.Kubernetes()
-		if err := kube.DeleteNodePool(clusterID, poolID); err != nil {
-			return err
-		}
-	} else {
+
+	if !(force || AskForConfirm("delete this Kubernetes node") == nil) {
```

Comment:
Wonder if we should call out here (when `replace` is false) or in the help that this operation will permanently resize the pool. This might be a bit confusing since we can also resize pools via the `node-pool update` command.

Reply:
More info is generally better with these sorts of confirmations. Especially in a case like this, wherein the number of nodes remaining in the pool could affect the response.